### ISSUE TITLE
Development server returned response error 500

### ISSUE THREAD

**Issue by squabbi** [SDPSMS/ManageU#11]
```
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `E:\Work\GitHub\ManageU\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (E:\Work\GitHub\ManageU\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:167:1306)
    at ResolutionRequest.resolveDependency (E:\Work\GitHub\ManageU\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:80:16)
    at DependencyGraph.resolveDependency (E:\Work\GitHub\ManageU\node_modules\metro\src\node-haste\DependencyGraph.js:237:485)
    at Object.resolve (E:\Work\GitHub\ManageU\node_modules\metro\src\lib\transformHelpers.js:116:25)
    at dependencies.map.result (E:\Work\GitHub\ManageU\node_modules\metro\src\DeltaBundler\traverseDependencies.js:298:29)
    at Array.map (<anonymous>)
    at resolveDependencies (E:\Work\GitHub\ManageU\node_modules\metro\src\DeltaBundler\traverseDependencies.js:294:16)
    at E:\Work\GitHub\ManageU\node_modules\metro\src\DeltaBundler\traverseDependencies.js:159:33
    at Generator.next (<anonymous>)
    at step (E:\Work\GitHub\ManageU\node_modules\metro\src\DeltaBundler\traverseDependencies.js:239:307)
```

This is what I got up to last time I tried running it on Android.

**Comment by squabbi** [comment]
Downgrading to react-native version 0.55.x gave a different error:

```
error: bundling failed: Error: Plugin 0 specified in "E:\\Work\\GitHub\\ManageU\\node_modules\\babel-preset-react-native\\index.js" provided an invalid property of "default" (While processing preset: "E:\\Work\\GitHub\\ManageU\\node_modules\\babel-preset-react-native\\index.js")
    at Plugin.init (E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\plugin.js:131:13)
    at Function.normalisePlugin (E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\file\options\option-manager.js:152:12)
    at E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\file\options\option-manager.js:184:30
    at Array.map (<anonymous>)
    at Function.normalisePlugins (E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\file\options\option-manager.js:158:20)
    at OptionManager.mergeOptions (E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\file\options\option-manager.js:234:36)
    at E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\file\options\option-manager.js:265:14
    at E:\Work\GitHub\ManageU\node_modules\babel-core\lib\transformation\file\options\option-manager.js:323:22
    at Array.map (<anonymous>)
```

Related to the babble pluggin

**Comment by squabbi** [comment]
Looking to update babel to use 7.0.0-beta in package.json
https://github.com/storybooks/storybook/issues/3897

**Comment by squabbi** [comment]
Everything complies and app connects to the server, however a new message is here now:

![image](https://user-images.githubusercontent.com/1893572/45251210-9fee0a80-b385-11e8-99c4-c073c2324583.png)
